### PR TITLE
Link Big File handling section from nginx chapter in the existing uploading files >10MB section

### LINKED ISSUE #194: nginx example configs: minor suggestions/ideas
Reported by @voidzero in https://github.com/nextcloud/server/issues/1552

### PR DESCRIPTION
**Pull request by joshtrichards**
### ☑️ Resolves

* Fix #194 

### 🖼️ Screenshots

image

### CODE REVIEW

File: admin_manual/installation/nginx.rst [hunk 1]
```diff
@@ -112,6 +112,8 @@ to the following to re-allow file uploads:
 
 See `issue #8802 on nextcloud/server <https://github.com/nextcloud/server/issues/8802>` for more information.
 
+Other parameters besides the above are relevant to uploading large files (see :doc:`../configuration_files/big_file_upload_configuration`).
```
Comment (ChristophWurst) [inline admin_manual/installation/nginx.rst:115]: Create an anchor and use :ref: so the link doesn't break when we move pages around https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor

**Comment by kesselb** [comment]
@joshtrichards could you update the pr as suggested?

**Comment by kesselb** [comment]
- Edit https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_files/big_file_upload_configuration.rst and add `.. _uploading_big_files:` before the heading (that's adding a label).

- Change ```:doc:`../configuration_files/big_file_upload_configuration` ``` to ```:ref:`uploading_big_files` ``` to use the label.

Thanks :+1:

**Comment by joshtrichards** [comment]
Yeah... Apologies to you both. It's been one of those weeks (couple actually). I'll clean up this branch... (that I merged master into inadvertently... twice :rofl:) then will address the actual change.

Need to finish setting up my new workspace on my new workstation... Was planning to get an actual Sphinx dev environment up first (and finally). Hoping to do it in a Docker container rather than install stuff locally or in a VM so created a little extra upfront work for myself.

**Review by susnux — APPROVED** [review]

**Pull request merged**